spm_results_export.m

```diff
@@ -236,8 +236,23 @@
 
 %-Display webpage
 %==========================================================================
-if(multipleExcursions && exNo >1)
-    [~, webID] = web(fHTML, '-new');
+%Check if we are running from the commandline. If no information about
+%cmdline is stored we are running code without running SPM (i.e. for 
+%tests). We display only if we are not in commandline mode.

```

Comment:
Is there really no dependency on SPM in the codebase?

Reply:
Ah, sorry this wasn't very clear! What I mean by this is that when we run the tests, we don't physically have SPM open and because of that sometimes the defaults aren't set (e.g. like with the previous comment it doesn't know the modality is FMRI and this can cause errors). I shall change this comment from 'running SPM' to 'having SPM open' to make it clearer.